test/cri-containerd/runpodsandbox_test.go

```diff
@@ -71,6 +77,63 @@ func Test_RunPodSandbox_LCOW(t *testing.T) {
 	runPodSandboxTest(t, request)
 }
 
+func Test_RunPodSandbox_Events_LCOW(t *testing.T) {
+	client := newTestRuntimeClient(t)
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+	podctx, podcancel := context.WithCancel(context.Background())
+	defer podcancel()
+
+	pullRequiredLcowImages(t, []string{imageLcowK8sPause})
+
+	request := &runtime.RunPodSandboxRequest{
+		Config: &runtime.PodSandboxConfig{
+			Metadata: &runtime.PodSandboxMetadata{
+				Name:      t.Name(),
+				Uid:       "0",
+				Namespace: testNamespace,
+			},
+		},
+		RuntimeHandler: lcowRuntimeHandler,
+	}
+
+	topicNames, filters := getTargetRunTopics()
+	targetNamespace := "k8s.io"
+
+	eventService := newTestEventService(t)
+	stream, errs := eventService.Subscribe(ctx, filters...)
+
+	podID := runPodSandbox(t, client, podctx, request)
+	stopAndRemovePodSandbox(t, client, podctx, podID)
```

Comment:
For this case you really do want to say `run`, `stop`, `remove` to verify the events after. So you dont need to create another method just call them in order

Reply:
This may have been leftover from the previous design. I will double check and remove if its not necessary :)